service/controller/legacy/v29/ebs/error.go

```diff
@@ -40,3 +44,7 @@ func IsVolumeNotFound(err error) bool {
 
 	return false
 }
+
+var volumeAttachedError = &microerror.Error{
```

Comment:
Let's please always have custom errors and their matchers together as described in https://github.com/giantswarm/fmt/blob/master/go/errors.md. As mentioned in the other comment above, we need the matching in the calling resource anyway.